app/black_spots/tasks/generate_training_input.py

```diff
@@ -0,0 +1,355 @@
+import logging
+import csv
+import os
+import tempfile
+from functools import partial
+from dateutil import parser
+from dateutil.relativedelta import relativedelta
+import pyproj
+import rtree
+import pytz
+import fiona
+import tarfile
+import shutil
+from collections import namedtuple
+from django.core.files import File
+
+from celery import shared_task
+from celery.utils.log import get_task_logger
+
+from shapely.geometry import mapping, shape, LineString, MultiPoint, Point
+from shapely.ops import transform
+
+from black_spots.models import (
+    BlackSpotRecordsFile, RoadSegmentsShapefile, BlackSpotTrainingCsv, ForecastTrainingCsv
+)
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger()
+logger = get_task_logger(__name__)
+
+INTERSECTION_BUFFER_UNITS = int(os.getenv('INTERSECTION_BUFFER_UNITS', '5'))
```

Comment:
It looks to me like a few of these are never used. My linter isn't complaining but a search for `INTERSECTION_BUFFER_UNITS` for example, only turns up the two matches on this line. Same for a few of these other constants.